mmv1/templates/terraform/examples/workstation_config_basic.tf.erb

```diff
@@ -0,0 +1,48 @@
+data "google_project" "project" {
+  provider = google-beta
+}
```

Comment:
Are these data sources used in the examples? If not, I think you should remove them from the examples to avoid confusion and reduce the size of the example blocks [in the generated documentation](https://github.com/modular-magician/terraform-provider-google-beta/blob/13b7d73767b3ff56fdc482ef6e295e23b24021c6/website/docs/r/workstations_workstation_config.html.markdown).

Reply:
Indeed not used, removed it.